packages/components/src/Badge/Badge.stories.js

```diff
@@ -4,6 +4,8 @@ import { action } from '@storybook/addon-actions';
 import talendIcons from '@talend/icons/dist/react';
 
 import Badge from './Badge.component';
+import FilterBar from '../FilterBar';
```

Comment:
Where is the FilterBar used in the story ? (alongside with the `talend-search` icon)
In the story I don't see any Filter bar

Reply:
Added a separate column for Badge with dropdowns, please check badge with custom items